caper/caper_args.py

```diff
@@ -5,7 +5,7 @@
 from autouri import URIBase
 
 from .arg_tool import update_parsers_defaults_with_conf
-from .backward_compatibility import CAPER_1_0_0_PARAM_KEY_NAME_CHANGE
+from .backward_compatibility import PARAM_KEY_NAME_CHANGE
```

Comment:
You are almost too nice to the users who won't upgrade :)

Reply:
Yes, this is actually because of a configuration file `~/.caper/default.conf`. Users don't read release notes and hence parameter name changes. If parameters were defined in the command line argument then it would errors out so that users would notice the change but any old parameters defined in a configuration file will be just ignored and take defaults values.